blocks/code/src/dev.tsx

```diff
@@ -2,10 +2,10 @@
  * This is the entry point for developing and debugging.
  * This file is not bundled with the library during the build process.
  */
-import { render } from "react-dom";
-import { tw } from "twind";
+import { createRoot } from "react-dom/client";
```

Comment:
Changes in this file have been copied from the latest version of `create-block-app`.